ParisBNF/abb/BNFabb205.xml

```diff
@@ -112,8 +112,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <persName ref="PRS1071dAbbadi">Antoine d'Abbadie</persName>
                      left Ethiopia in 1848. It is not clear if <foreign xml:lang="gez">
                      ጽላ፡ መስቀል፡</foreign> in <ref target="#a1">Addition 1</ref> refers to a person who
-                   owned the book, or to a church or monastery with a <foreign xml:lang="gez">tābot</foreign>
-                 dedicated to the True Cross (<foreign xml:lang="gez">Masqal</foreign>).</note>
+                   owned the book or to a church or monastery with a <foreign xml:lang="gez">tābot</foreign>
+                   (<foreign xml:lang="gez">ṣəlla</foreign>)
+                 dedicated <foreign xml:lang="gez">Masqal</foreign>, i.e.
+                  <ref type="authFile" corresp="Masqal">the True Cross</ref>.</note>
```

Comment:
this way for encoding a tabot was actually meant for `<ab type="tabot"> ` within a place/institution record, so that we could filter for tabots
since there is no `<ab type="tabot">` in a MS-record, this encoding does not provide the information that we are speaking here about a tabot
I wonder if we can introduce adding (especially in such cases, where we have no record for the institution, but want to be able to find it when we search for all altars dedicated to the Cross or similar things) a `@subtype` "tabot" to the `<ref type="authFile">` - @PietroLiuzzo

Reply:
ok